tic_tac_toe/tic_tac_toe.py

```diff
@@ -0,0 +1,13 @@
+class TicTacToe:
```

Comment:
Sounds good! I've been trying to think about design more and I was wondering that too: what are the responsibilities of my classes? Should I have multiple classes? It would be great to chat about it in more detail.